src/ex_cmds2.c

```diff
@@ -457,6 +457,33 @@ ex_listdo(exarg_T *eap)
     tabpage_T	*tp;
     buf_T	*buf = curbuf;
     int		next_fnum = 0;
+
+    if (curwin->w_p_stb)
+    {
+      if (eap->cmdidx == CMD_ldo && !eap->forceit)
+      {
+        // Disallow :ldo if 'stickybuf' is applied
+        semsg(_("E922: Cannot edit buffer. 'switchbuf' is enabled. Use ! to force it."));
```

Comment:
Use a different error number here please, or unify the error messages to be always the same

Reply:
I don't know the history or conventions of Vim error codes but if `E1513`+ is okay, I'll append to those.

I tried to find an existing gap in the error codes but the only 3 contiguous error codes that were open are `E1045-E1046` and `E1412-E1499`, which appear to be reserved for Vim9 related things. And `E56-E59` too, I suppose